openmdao/visualization/n2_viewer/tests/gui_test_models/turbojet.py

```diff
@@ -0,0 +1,30 @@
+import openmdao.api as om
```

Comment:
This is a simple paraboloid model, not a turbojet model, so the filename should be changed.